certified-connectors/sms77io/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this API",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#11D472",
+    "capabilities": [],
+    "publisher": "Andre Matthies"
```

Comment:
The `publisher` field shouldn't contain any PII. So it can be changed to `sms77 e.K.` (or similar).

Related to that, the `stackOwner` field appears to be missing. For reference, this field is discussed in the docs found [here](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#publisher-and-stack-owner). It can also be `sms77 e.K.` (or similar)

Reply:
Whoops, I missed that one.
It should be fixed now!